packages/design-picker/src/available-designs-config.json

```diff
@@ -424,6 +424,21 @@
 			"is_premium": false,
 			"is_fse": true,
 			"features": []
+		},
+		{
+			"title": "Start with an empty page",
+			"slug": "blank-canvas",
+			"template": "blank-canvas",
+			"theme": "blank-canvas",
+			"fonts": {
+				"headings": "Source Sans Pro",
+				"base": "Source Sans Pro"
+			},
```

Comment:
Shouldn't we skip passing font values here to let the theme use its defaults?

Reply:
> Assuming #52107 will be solved
Yes. For now I kept the values there, so that the flow keeps working locally. The plan is to keep this PR as draft until  #52107 is solved — I mostly opened this PR early to get the conversation started on https://github.com/Automattic/wp-calypso/pull/52113#discussion_r617070323